src/app/subscribe/page.jsx

```diff
@@ -33,7 +33,7 @@ const PackageCard = ({ pkg, index, highlightColor }) => {
     return (
         <StyledCard>
             <StyledImageContainer highlightColor={highlightColor} theme={theme}>
-                <img src={`/sub${index + 1}.png`} alt={`${pkg.name} image`} style={{ width: '460px', height: '270px', objectFit: 'cover', boxShadow: theme.palette.mode === 'light' ? '0px 4px 30.1px rgba(0, 0, 0, 0.5)' : 'none', }} />
+                <img src={`/bookingFormIcons/sub${index + 1}.png`} alt={`${pkg.name} image`} style={{ width: '460px', height: '270px', objectFit: 'cover', boxShadow: theme.palette.mode === 'light' ? '0px 4px 30.1px rgba(0, 0, 0, 0.5)' : 'none', }} />
```

Comment:
why are you using style here? also, try to move the style to a separate file if its more than 4 5 lines in the whole code. contact Faizan if you need any help